examples/crate.elm

```diff
@@ -143,42 +170,120 @@ view { texture, theta } =
             []
 
         Just tex ->
-            [ render vertexShader fragmentShader crate { crate = tex, perspective = perspective theta } ]
+            let
+                camera =
+                    perspective theta
+            in
+                [ renderBox
+                    [ disable stencilTest ]
```

Comment:
This one is supposed to be called at the end of the pipeline.
With the current api this would require a dummy render call at the end, or as I did here calling it at the beginning.
Here a `withStencil : List Renderable -> Renderable would be nice`